src/drawer.canvas.js

```diff
@@ -100,6 +119,61 @@ WaveSurfer.util.extend(WaveSurfer.Drawer.Canvas, {
         }, this);
     },
 
+    drawWaveMainWaveform: function (peaks, length, max) {
+    // A half-pixel offset makes lines crisp
+    var $ = 0.5 / this.params.pixelRatio;
+
+    var vscale = 0.95; // scaled values of height of waveform
```

Comment:
For stereo channels, the waveform would stack on top of each other. If both channels are on full blast, I thought it would have been better to add some spacing between the waveforms. So I scaled down the waveform height to 0.95 times of its original height.